config/initializers/spotlight_initializer.rb

```diff
@@ -110,13 +110,17 @@
 #   'my-local-config': ->(context) { context.my_custom_data_path(context.current_exhibit) }
 # }
 
-Spotlight::Engine.config.exhibit_themes = %w[default example joyce]
+Spotlight::Engine.config.exhibit_themes = %w[default example dr-joyce-brothers blackprint]
 
 Exhibits::Application.config.after_initialize do
   Spotlight::Exhibit.themes_selector = ->(exhibit) do
     default_themes = %w[default example]
-    if exhibit.slug == 'dr-joyce-brothers'
-      [*default_themes, 'joyce']
+    custom_themes = %w[dr-joyce-brothers blackprint]
```

Comment:
Looks like the selected exhibit theme is stored in the db - do you know what happens to an exhibit if it's no longer associated with a valid theme? Not sure if we'll be able to just go into the Exhibit Dashboard to update the joyce exhibit's theme, or if we'll have to go into the rails console. Either should be quick though, so not a huge deal.

Reply:
Phew! That seems very reasonable though.